sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -262,8 +262,8 @@ public virtual async Task<Response<UploadLogsResult>> UploadAsync<T>(string rule
             Argument.AssertNotNullOrEmpty(logs, nameof(logs));
 
             // Calculate the number of threads to use.
-            // If there are 0 workers or an UploadLogsOptions object was not provided, method will run serially. Otherwise will run in parallel with number of workers given.
-            int _maxWorkerCount = (options == null || options.MaxConcurrency <= 0) ? DefaultWorkerCount : options.MaxConcurrency;
+            // If there are 0 workers or an UploadLogsOptions object was not provided, method will run will 5 threads. Otherwise will run in parallel with number of workers given.
```

Comment:
```suggestion
            // If there are 0 workers or an UploadLogsOptions object was not provided, method will run with 5 workers. Otherwise will run in parallel with number of workers given.
```